VirtualizingWrapPanel/VirtualizingWrapPanel/VirtualizingWrapPanel.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.0;net45</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net45</TargetFrameworks>
```

Comment:
.NET Core 3.0 support should not be dropped, please target 3.0 and 3.1

Reply:
Done